### DISCUSSION TITLE
is it possible to use this as a util?

### DISCUSSION THREAD

**Discussion by hulkish** [haraka/Haraka#3264] [Q&A]
As opposed to running a full fledged SMTP server... is it possible to use this as a util to just sent an email - from say just a scripting perspective?

For example, php has.a.way of sending emails via a  `mail()` function - without needing to setup any SMTP server. Is that possible with haraka? If so, please point me in the right direction.

**Comment by msimerson** [comment]
Yes, but you should likely be using nodemailer instead.
